src/ParametersHandler/include/BipedalLocomotion/ParametersHandler/IParametersHandler.h

```diff
@@ -181,7 +188,17 @@ class IParametersHandler
      * @warning Please implement the specific version of this method in the Derived class. Please
      * check YarpImplementation::getGroup
      */
-    virtual weak_ptr getGroup(const std::string& name) const = 0;
+    virtual const_weak_ptr getGroup(const std::string& name) const = 0;
+
+    /**
+     * Get a Group from the handler.
+     * @param name name of the group
+     * @return A pointer to IParametersHandler, if the group is not found the weak pointer cannot
+     * be locked
+     * @warning Please implement the specific version of this method in the Derived class. Please
+     * check YarpImplementation::getGroup
+     */
+    virtual weak_ptr getGroup(const std::string& name) = 0;
```

Comment:
The same as for the other implementations. I would suggest adding a comment on the fact that the output group can be edited.